index.html

```diff
@@ -338,7 +263,59 @@ <h4>I'm always interested in hearing about new projects,
       <img class="bottom-line" src="images/Shape.png" alt="black line">
     </section>
   </div> 
+   <!-- desktop version-->
+   <section class="pop-windows">
+    <div class="container">
+      <h1>Keeping track of hundreds of components website</h1>
+      <img class="exit-sign" id="exit-sign-d" src="images/Icons/ex.png" alt="exit-sign">
+      <img class="tags" src="images/Icons/Tags.png" alt="tags">
+      <div class="middle">
+        <img class="snapshot" src="images/Snapshoot-Portfolio-1.png" alt="snapshot picture">
+        <aside>
+          <p>Lorem Ipsum is simply dummy text of the printing and
+             typesetting industry. Lorem Ipsum has been the
+             industry's standard dummy text ever since the 1500s,
+             when an unknown printer took a galley of type and
+             scrambled it 1960s with the releaLorem Ipsum is simply
+             dummy text of the printing and typesetting industry.
+             Lorem Ipsum has been the industry's standard dummy
+             text ever since the 1500s, when an unknown printer
+             took a galley of type and scrambled it 1960s with
+             the releax map lapora verita.
+          </p>
+          <button type="button" class="see-live-btn"></button>
+          <button type="button" class="see-source-btn"></button>
+        </aside>
+      </div>
+    </div>
 
+    <!-- mobile version-->
+    <div class="mobile-container">
+        <h1>Multi Post Stories</h1>
+        <img class="exit-sign" id="exit-sign-m" src="images/Icons/ex.png" alt="exit-sign">
+        <img class="tags" src="images/Icons/Tags.png" alt="tags">
+        <div class="middle">
+        <img class="snapshot" src="images/Snapshoot-Portfolio.png" alt="snapshot picture">
+
+        <p>Lorem Ipsum is simply dummy text of the printing and
+            typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s,
+            when an unknown printer took a galley of type and
+            scrambled it 1960s with the releaLorem Ipsum is simply
+            dummy text of the printing and typesetting industry.
+            Lorem Ipsum has been the industry's standard dummy
+            text ever since the 1500s, when an unknown printer
+            took a galley of type and scrambled it 1960s with
+            
+            the releax map lapora verita.
+        </p>
+        <div class="buttons">
+          <button type="button" class="see-live-btn"></button>
+          <button type="button" class="see-source-btn"></button>
+        </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Please Modal popup data is populated with the data from the projects array. We don't need to see the popup code inside HTML.
  Kindly remember your [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md): In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects...

- [ ] Kindly make sure the popup is responsive, it fits mobile and desktop sizes properly